dpu-cni/pkgs/sriov/sriov.go

```diff
@@ -17,6 +17,10 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
+const (
+	host_vlans = false
```

Comment:
Add: // TODO: Some vendors have issues with Host VLANs, thus disable the feature for now